image/image_linux.go

```diff
@@ -723,5 +811,12 @@ var setupSeed = func(tsto *tooling.ToolingStore, model *asserts.Model, opts *Opt
 			return err
 		}
 	}
-	return nil
+
+	// TODO: There will be classic UC20+ model based systems
+	//       that will have a bootable  ubuntu-seed partition.
+	//       This will need to be handled here eventually too.
+	if opts.Classic {
+		return is.finishSeedClassic()
+	}
+	return is.finishSeedCore(opts)
```

Comment:
and these could be organized into a imageSeeder.Finish method

Reply:
Done!